### PR TITLE
HATCH-349/add client side functions

### PR DESCRIPTION
**Pull request by CMLabay**
# Description

# Jira ticket links

# Test Plan

# Definition of done

- [ ] Does new code have 90% testing coverage (100% for `core`) of both unit and E2E tests?
- [ ] Is code secure? If applicable, add security notes to the description.
- [ ] Do all new TODO comments have Jira links with enough information?
- [ ] Were all changes published and deployed to [the grid](https://github.com/bitovi/hatchify-grid-demo)?
- [ ] Has the browser error-console been reviewed to not contain new errors introduced by these code changes?
- [ ] The site looks “good” above 1000px width.
- [ ] The site looks “good” when the window height is small. No double scrollbars.
- [ ] Were the changes tested to ensure 508 compliance?

### CODE REVIEW

File: packages/core/src/dataTypes/enumerate/getFinalize.ts [hunk 1]
```diff
@@ -51,7 +95,7 @@ export function getFinalize(
     // Example : ?filter[age]=xyz ... xyz => throw "xyz is not a string";
     setORMQueryFilterValue: (queryValue: string): string | null => {
       if (["null", "undefined"].includes(queryValue)) {
-        if (control.allowNull !== false) {
+        if (control.allowNull) {
```
Comment (arthurpankiewicz) [inline packages/core/src/dataTypes/enumerate/getFinalize.ts:98]: are you sure we need to change this? (backend functions)
i think allowNull can be `undefined|boolean` so should be careful removing this check for explicit false

File: packages/core/src/dataTypes/string/getFinalize.ts [hunk 2]
```diff
@@ -95,7 +95,7 @@ export function getFinalize(
     // Example : ?filter[age]=xyz ... xyz => throw "xyz is not a string";
     setORMQueryFilterValue: (queryValue: string): string | null => {
       if (["null", "undefined"].includes(queryValue)) {
-        if (control.allowNull !== false) {
+        if (control.allowNull) {
```
Comment (arthurpankiewicz) [inline packages/core/src/dataTypes/string/getFinalize.ts:98]: same question here

File: packages/core/src/dataTypes/enumerate/getFinalize.ts (hunk 1 above)
Comment (CMLabay) [reply packages/core/src/dataTypes/enumerate/getFinalize.ts:98]: That's fair

**Review by arthurpankiewicz — APPROVED** [review]
👍

**Pull request merged**